etl/steps/data/garden/unodc/2025-05-12/justice.meta.yml

```diff
@@ -2,9 +2,11 @@
-      - Even though these differences exist, the data is still useful for understanding how criminal justice systems work in different countries.
-      - The data is collected from official records kept by different parts of the justice system. The police usually record information about people who are suspected of, arrested for, or cautioned for a crime. Prosecutors collect data about people being charged, and the courts collect information about people who appear before them and those who are convicted.
+        - Bribery cases progress slowly through justice systems, creating significant timing mismatches in the data. Individuals arrested in one year are typically prosecuted the following year and may not be convicted until several years later. This extended timeline means arrest and conviction data from the same year reflect entirely different groups of cases at different stages of the process, making direct comparisons misleading.
+        - Each component of the justice system maintains separate recording systems that capture information at different stages and using different criteria. Police document cases when investigations begin, prosecutors when formal charges are filed, and courts when cases are heard and decided. These disconnected systems create data gaps that complicate efforts to track individual cases through the entire process from arrest to final disposition.
+        - Bribery cases present unique prosecution challenges due to their inherent complexity. These crimes frequently involve sophisticated criminal networks, organized crime connections, and require specialized investigative resources and expertise that many justice systems lack. This complexity results in higher case attrition rates compared to conventional crimes, with more cases being dropped or dismissed at various stages of the process.
+        - Countries employ different approaches to data collection, classification, and reporting that affect comparability. Some nations implement the "principal offence rule" where only the most serious crime is counted when multiple offenses occur together. Data completeness varies significantly across countries, and adherence to international reporting standards like the [International Classification of Crime for Statistical Purposes (ICCS)](https://www.unodc.org/unodc/en/data-and-analysis/statistics/iccs.html) remains inconsistent, leading to definitional and methodological differences.
+        - Apparent anomalies in the data, such as prosecution rates exceeding arrest rates in certain countries, typically reflect these systematic data collection challenges rather than actual procedural irregularities.
```

Comment:
There are quite a few caveats here. Do you think the data still shows something that can be understood by the public? I wonder which of arrests, prosecutions, or convictions is likely to have the most solid data collection; it could be worth just focusing on the best variable rather than comparing across them.